### PR TITLE
Update registry.credentials.existingSecret comment

### PR DESCRIPTION
**Pull request by jameshearttech**
I have updated the comment for registry.credentials.existingSecret based on my personal experience.

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -590,7 +590,7 @@ registry:
   credentials:
     username: "harbor_registry_user"
     password: "harbor_registry_password"
-    # If using existingSecret, the key must be REGISTRY_PASSWD and REGISTRY_HTPASSWD
+    # Secret keys must be REGISTRY_PASSWD and REGISTRY_HTPASSWD. For key REGISTRY_PASSWD the value is the password. For key REGISTRY_HTPASSWD the value is the string in the password file generated by htpasswd where the username is harbor_registry_user and the encryption type is bcrypt. For example: `htpasswd -bBc passwordfile harbor_registry_user harbor_registry_password`. The username must be harbor_registry_user!
```
Comment (ywk253100) [inline values.yaml:593]: Could you also update the description in README.md?

Comment (jameshearttech) [reply values.yaml:593]: Yes, I'll do that.

Comment (jameshearttech) [reply values.yaml:593]: I updated README.md. Lmk if you need anything else. Thanks!

**Comment by zyyw** [comment]
Hi @jameshearttech, thank you for contributing to harbor-helm. Could you please do:
* resolve the conflict 
* add commit signature to resolve **DCO** error in the CI

Thanks again!

**Comment by zyyw** [comment]
closing the PR due to inactivity.

**Pull request closed without merging**